### PR TITLE
Broken SVG Links

### PR DESCRIPTION
**Pull request by derappelt**
Svg anchor tags return an object( {baseVal:string, animVal:string} ) instead of an string. 
This causes react-snap to throw an error

### CODE REVIEW

**Comment by stereobooster** [comment]
Hi. Thank you for contribution. Can you please add test-case for this?

**Comment by derappelt** [comment]
Does this test cover enough for you?

**Comment by stereobooster** [comment]
Oh yes, sorry. I was distracted. Everything looks fine, I just want to run some tests, to make sure I understand what is happening.

**Comment by derappelt** [comment]
Thanks no problem! 
I'm not quite sure if my submitted code covers all SVG Link cases. But it works in all projects I tried so far.
Like I mentioned in my comment above, I think the first priority should be that Inline SVG Links don't crash snap.

File: src/puppeteer_utils.js [hunk 1]
```diff
@@ -91,7 +91,12 @@ const enableLogging = opt => {
 const getLinks = async opt => {
   const { page } = opt;
   const anchors = await page.evaluate(() =>
-    Array.from(document.querySelectorAll("a")).map(anchor => anchor.href)
+    Array.from(document.querySelectorAll("a")).map(anchor => {
+      if(anchor.href.baseVal){
+        return location.origin + anchor.href.baseVal;
```
Comment (stereobooster) [inline src/puppeteer_utils.js]:
```suggestion
        const a = document.createElement("a");
        a.href = anchor.href.baseVal;
        return a.href;
```

**Comment by stereobooster** [comment]
There is small thing to change, but I can do it. Thanks

**Pull request merged**